### PR TITLE
feat(discover): Add count_web_vitals field to Query Builder

### PR DESCRIPTION
**Pull request by 0Calories**
This PR adds a new function field to Discover's query builder: `count_web_vitals()`

image

### CODE REVIEW

File: static/app/utils/discover/fields.tsx [hunk 1]
```diff
@@ -212,6 +255,33 @@ export const AGGREGATIONS = {
...
+      },
+      {
+        kind: 'dropdown',
+        options: WEB_VITALS_QUALITY,
+        dataType: 'string',
+        defaultValue: WEB_VITALS_QUALITY[0].value,
+        required: true,
+      },
+    ],
+    documentation: t('web vitals threshold counts'),
```
Comment (wmak) [inline static/app/utils/discover/fields.tsx:280]:
```suggestion
    documentation: t('events matching vital thresholds'),
```

File: static/app/utils/discover/fields.tsx [hunk 2]
```diff
@@ -693,30 +763,6 @@ export function formatTagKey(key: string): string {
...
-  FramesSlow = 'measurements.frames_slow',
-  FramesFrozen = 'measurements.frames_frozen',
-  FramesSlowRate = 'measurements.frames_slow_rate',
-  FramesFrozenRate = 'measurements.frames_frozen_rate',
-  StallCount = 'measurements.stall_count',
-  StallTotalTime = 'measurements.stall_total_time',
-  StallLongestTime = 'measurements.stall_longest_time',
-  StallPercentage = 'measurements.stall_percentage',
-}
-
```
Comment (wmak) [inline static/app/utils/discover/fields.tsx:719]: Was this moved cause these needed to be declared before `AGGREGATIONS`?

**Review by wmak — APPROVED** [review]
lgtm

File: static/app/utils/discover/fields.tsx (hunk 2 above)
Comment (0Calories) [reply static/app/utils/discover/fields.tsx:719]: Yeah exactly, I only needed to move the `WebVital` enum but I thought I should move `MobileVital` as well for the sake of consistency

**Pull request merged**